cluster/src/main/java/org/apache/iotdb/cluster/ClusterMain.java

```diff
@@ -179,10 +180,11 @@ private static void startServerCheck() throws StartupException {
     // assert this node is in seed nodes list
     Node localNode = new Node();
     localNode
-        .setIp(config.getClusterRpcIp())
+        .setIp(config.getInternalIp())
         .setMetaPort(config.getInternalMetaPort())
         .setDataPort(config.getInternalDataPort())
-        .setClientPort(config.getClusterRpcPort());
+        .setClientPort(config.getClusterRpcPort())
```

Comment:
We should split the cluster_rpc_port and the rpc_port due to the following reasons, and because the cluster_rpc_port have been Initializatiioned, so it's actually cluster_rpc_port

the rpc_port of a single module can not be used directly, because when the debug mode(open_server_rpc_port=true) is turned on, the rpc_port of a single module may be different from that of a distributed cluster.

Reply:
got it